examples/kubernetes-yaml/splunk-otel-collector-gateway.yaml

```diff
@@ -137,10 +137,16 @@ spec:
     port: 13133
   - name: jaeger-grpc # Default endpoint for Jaeger gRPC receiver.
     port: 14250
-  - name: jaeger-thrift # Default endpoint for Jaeger Thrift receiver.
+  - name: thrift-binary # Default endpoint for Jaeger Thrift receiver.
+    port: 6832
```

Comment:
Are these needed for gateway mode? They are udp

Reply:
Yes, otherwise ports are not exposed on the k8s service. If UDP I need to update protocol then

Reply:
updated

Reply:
Does removing the jaeger prefix shift some guessing work to the user?  `jaeger-thrift-binary`, etc.

Reply:
K8s has a limit for port names - it must be less then 16 characters

Reply:
This is gateway mode -- why would udp ports be required?

Reply:
It’s not required but nice to have if user wants to send traces directly to gateway I guess. The PR is just to update service deployment to what’s already configured on the gateway. We can either change the gateway config - remove jaeger receivers or expose ports for them in the service deployment

Reply:
just to clarify: the getaway already has the binary thrift receiver configured, but the port is not enabled on the service deployment - this PR fixes it